src/labkey/security/Permission.ts

```diff
@@ -384,13 +384,8 @@ export function hasEffectivePermission(effectivePermissions: string[], desiredPe
 }
 
 /**
- * Returns 1 if the permission passed in 'perm' is on in the permissions
- * set passed as 'perms'. This is a local function and does not make a call to the server.
- * @param perms The permission set, typically retrieved for a given user or group.
- * @param perm A specific permission bit to check for.
+ * @deprecated Do not use! Use hasEffectivePermission() instead. This will be removed soon.
```

Comment:
You're more emphatic here than elsewhere. I guess you really don't like this method.